.buildkite/metricbeat/pipeline.yml

```diff
@@ -57,7 +57,7 @@ steps:
           # defines the MODULE env var based on what's changed in a PR
           source .buildkite/scripts/changesets.sh
           defineModuleFromTheChangeSet metricbeat
-          echo "~~~ Will run tests with env var MODULE=$$MODULE"
```

Comment:
We remove references to the env var `MODULE` because if unset, it fails the command due to undefined variable checks (`set -e)`